diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..dff86a5 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -30,6 +30,21 @@ set_target_properties(rnscreens PROPERTIES
     POSITION_INDEPENDENT_CODE ON
 )
 
+# FLAGS DE COMPILACIÓN (para C y C++)
+target_compile_options(rnscreens PRIVATE
+    "-ffile-prefix-map=${CMAKE_SOURCE_DIR}=."
+)
+
+# FLAGS DE LINKER
+target_link_options(rnscreens PRIVATE
+    "-Wl,--build-id=none"
+    "-Wl,--no-relax"
+    "-Wl,--pack-dyn-relocs=none"
+    "-Wl,--sort-common"
+    "-Wl,--sort-section=name"
+    "-Wl,--hash-style=gnu"
+)
+
 target_compile_definitions(
     rnscreens
     PRIVATE
diff --git a/node_modules/react-native-screens/android/build.gradle b/node_modules/react-native-screens/android/build.gradle
index 872d44e..477a27d 100644
--- a/node_modules/react-native-screens/android/build.gradle
+++ b/node_modules/react-native-screens/android/build.gradle
@@ -132,6 +132,9 @@ android {
                 arguments "-DANDROID_STL=c++_shared",
                         "-DRNS_NEW_ARCH_ENABLED=${IS_NEW_ARCHITECTURE_ENABLED}",
                         "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
+                arguments "-DCMAKE_C_FLAGS=-ffile-prefix-map=${rootDir}=.",
+                        "-DCMAKE_CXX_FLAGS=-ffile-prefix-map=${rootDir}=.",
+                        "-DCMAKE_SHARED_LINKER_FLAGS=-Wl,--build-id=none,--no-relax,--pack-dyn-relocs=none,--sort-common,--sort-section=name,--hash-style=gnu"
             }
         }
     }
