diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..03f7529 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -30,6 +30,25 @@ set_target_properties(rnscreens PROPERTIES
     POSITION_INDEPENDENT_CODE ON
 )
 
+target_compile_options(
+    rnscreens 
+    PRIVATE
+    -ffile-prefix-map=${CMAKE_SOURCE_DIR}=.
+    -fno-ident
+    -g0
+)
+
+target_link_options(
+    rnscreens 
+    PRIVATE
+    -Wl,--build-id=none
+    -Wl,--no-relax
+    -Wl,--pack-dyn-relocs=none
+    -Wl,--sort-common
+    -Wl,--sort-section=name
+    -Wl,--hash-style=gnu
+)
+
 target_compile_definitions(
     rnscreens
     PRIVATE
diff --git a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
index 8502124..b6db9e4 100644
--- a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.13)
 set(CMAKE_VERBOSE_MAKEFILE ON)
+set(ENV{SOURCE_DATE_EPOCH} "1710000000")
 
 set(LIB_LITERAL rnscreens)
 set(LIB_TARGET_NAME react_codegen_${LIB_LITERAL})
@@ -20,6 +21,15 @@ add_compile_options(
   -Wno-dollar-in-identifier-extension
 )
 
+add_link_options(
+  -Wl,--build-id=none
+  -Wl,--no-relax
+  -Wl,--pack-dyn-relocs=none
+  -Wl,--sort-common
+  -Wl,--sort-section=name
+  -Wl,--hash-style=gnu
+)
+
 file(GLOB LIB_CUSTOM_SRCS CONFIGURE_DEPENDS *.cpp ${LIB_COMMON_COMPONENTS_DIR}/*.cpp ${LIB_COMMON_COMPONENTS_DIR}/utils/*.cpp)
 file(GLOB LIB_CODEGEN_SRCS CONFIGURE_DEPENDS ${LIB_ANDROID_GENERATED_COMPONENTS_DIR}/*.cpp)
 
@@ -76,6 +86,20 @@ target_compile_options(
   -frtti
   -std=c++20
   -Wall
+  -ffile-prefix-map=${CMAKE_SOURCE_DIR}=.
+  -fno-ident
+  -g0
+)
+
+target_link_options(
+  ${LIB_TARGET_NAME}
+  PRIVATE
+  -Wl,--build-id=none
+  -Wl,--no-relax
+  -Wl,--pack-dyn-relocs=none
+  -Wl,--sort-common
+  -Wl,--sort-section=name
+  -Wl,--hash-style=gnu
 )
 
 target_include_directories(
