diff --git a/node_modules/react-native-barcode-zxing-scan/android/build.gradle b/node_modules/react-native-barcode-zxing-scan/android/build.gradle
index cb55e00..5cc463e 100644
--- a/node_modules/react-native-barcode-zxing-scan/android/build.gradle
+++ b/node_modules/react-native-barcode-zxing-scan/android/build.gradle
@@ -12,12 +12,12 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 34
     buildToolsVersion = "27.0.3"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 30
+        minSdkVersion 24
+        targetSdkVersion 34
         versionCode 1
         versionName "1.0"
     }
