diff --git a/node_modules/react-native-reanimated/android/CMakeLists.txt b/node_modules/react-native-reanimated/android/CMakeLists.txt
index aff1f83..aaa6fc7 100644
--- a/node_modules/react-native-reanimated/android/CMakeLists.txt
+++ b/node_modules/react-native-reanimated/android/CMakeLists.txt
@@ -1,5 +1,7 @@
 project(Reanimated)
 cmake_minimum_required(VERSION 3.8)
+set(ENV{SOURCE_DATE_EPOCH} "1710000000")
+set(ENV{ZERO_AR_DATE} "1")
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
@@ -29,7 +31,7 @@ string(
 
 string(APPEND CMAKE_CXX_FLAGS
        " -fexceptions -fno-omit-frame-pointer -frtti -fstack-protector-all\
-   -std=c++${CMAKE_CXX_STANDARD} -Wall -Werror -ffile-prefix-map=${CMAKE_SOURCE_DIR}=. -fno-ident -g0 -gno-record-gcc-switches")
+   -std=c++${CMAKE_CXX_STANDARD} -Wall -Werror -ffile-prefix-map=${CMAKE_SOURCE_DIR}=. -fno-ident -g0 -gno-record-gcc-switches -frandom_seed=1")
 
 if(${IS_NEW_ARCHITECTURE_ENABLED})
   string(APPEND CMAKE_CXX_FLAGS " -DRCT_NEW_ARCH_ENABLED")
