diff --git a/node_modules/react-native-reanimated/android/CMakeLists.txt b/node_modules/react-native-reanimated/android/CMakeLists.txt
index 967272c..aff1f83 100644
--- a/node_modules/react-native-reanimated/android/CMakeLists.txt
+++ b/node_modules/react-native-reanimated/android/CMakeLists.txt
@@ -11,6 +11,15 @@ set(CMAKE_CXX_STANDARD 20)
 include("${REACT_NATIVE_DIR}/ReactAndroid/cmake-utils/folly-flags.cmake")
 add_compile_options(${folly_FLAGS})
 
+add_link_options(
+  -Wl,--build-id=none
+  -Wl,--no-relax
+  -Wl,--pack-dyn-relocs=none
+  -Wl,--sort-common
+  -Wl,--sort-section=name
+  -Wl,--hash-style=gnu
+)
+
 string(
   APPEND
   CMAKE_CXX_FLAGS
@@ -20,7 +29,7 @@ string(
 
 string(APPEND CMAKE_CXX_FLAGS
        " -fexceptions -fno-omit-frame-pointer -frtti -fstack-protector-all\
-   -std=c++${CMAKE_CXX_STANDARD} -Wall -Werror")
+   -std=c++${CMAKE_CXX_STANDARD} -Wall -Werror -ffile-prefix-map=${CMAKE_SOURCE_DIR}=. -fno-ident -g0 -gno-record-gcc-switches")
 
 if(${IS_NEW_ARCHITECTURE_ENABLED})
   string(APPEND CMAKE_CXX_FLAGS " -DRCT_NEW_ARCH_ENABLED")
