diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
index 2de3ca1..7c5421a 100644
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -257,6 +257,9 @@ android {
                         "-DIS_REANIMATED_EXAMPLE_APP=${IS_REANIMATED_EXAMPLE_APP}",
                         "-DREANIMATED_VERSION=${REANIMATED_VERSION}",
                         "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
+                arguments "-DCMAKE_C_FLAGS=-ffile-prefix-map=${rootDir}=.",
+                        "-DCMAKE_CXX_FLAGS=-ffile-prefix-map=${rootDir}=.",
+                        "-DCMAKE_SHARED_LINKER_FLAGS=-Wl,--build-id=none,--no-relax,--pack-dyn-relocs=none,--sort-common,--sort-section=name,--hash-style=gnu"
                 abiFilters (*reactNativeArchitectures())
                 targets("reanimated", "worklets")
             }
