diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..1382e05 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -1,4 +1,6 @@
 cmake_minimum_required(VERSION 3.9.0)
+set(ENV{SOURCE_DATE_EPOCH} "1710000000")
+set(ENV{ZERO_AR_DATE} "1")
 
 project(rnscreens)
 
@@ -30,6 +32,29 @@ set_target_properties(rnscreens PROPERTIES
     POSITION_INDEPENDENT_CODE ON
 )
 
+target_compile_options(
+    rnscreens 
+    PRIVATE
+    -ffile-prefix-map=${CMAKE_SOURCE_DIR}=.
+    -fno-ident
+    -g0
+    -gno-record-gcc-switches
+    -frandom-seed=1
+    -Wno-builtin-macro-redefined -D__DATE__= -D__TIME__= -D__TIMESTAMP__=
+    -no-canonical-prefixes
+)
+
+target_link_options(
+    rnscreens 
+    PRIVATE
+    -Wl,--build-id=none
+    -Wl,--no-relax
+    -Wl,--pack-dyn-relocs=none
+    -Wl,--sort-common
+    -Wl,--sort-section=name
+    -Wl,--hash-style=gnu
+)
+
 target_compile_definitions(
     rnscreens
     PRIVATE
diff --git a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
index 8502124..6fcd313 100644
--- a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
 set(CMAKE_VERBOSE_MAKEFILE ON)
+set(ENV{SOURCE_DATE_EPOCH} "1710000000")
+set(ENV{ZERO_AR_DATE} "1")
 
 set(LIB_LITERAL rnscreens)
 set(LIB_TARGET_NAME react_codegen_${LIB_LITERAL})
@@ -76,6 +78,13 @@ target_compile_options(
   -frtti
   -std=c++20
   -Wall
+  -ffile-prefix-map=${CMAKE_SOURCE_DIR}=.
+  -fno-ident
+  -g0
+  -gno-record-gcc-switches
+  -frandom-seed=1
+  -Wno-builtin-macro-redefined -D__DATE__= -D__TIME__= -D__TIMESTAMP__=
+  -no-canonical-prefixes
 )
 
 target_include_directories(
