diff --git a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
index 66bb972..633403e 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
 set(CMAKE_VERBOSE_MAKEFILE ON)
+set(ENV{SOURCE_DATE_EPOCH} "1710000000")
+set(ENV{ZERO_AR_DATE} "1")
 
 set(LIB_LITERAL safeareacontext)
 set(LIB_TARGET_NAME react_codegen_${LIB_LITERAL})
@@ -69,7 +71,6 @@ else()
   )
 endif()
 
-
 target_compile_options(
   ${LIB_TARGET_NAME}
   PRIVATE
@@ -78,6 +79,13 @@ target_compile_options(
   -frtti
   -std=c++20
   -Wall
+  -ffile-prefix-map=${CMAKE_SOURCE_DIR}=.
+  -fno-ident
+  -g0
+  -gno-record-gcc-switches
+  -frandom-seed=1
+  -Wno-builtin-macro-redefined -D__DATE__= -D__TIME__= -D__TIMESTAMP__=
+  -no-canonical-prefixes
 )
 
 target_include_directories(
