diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..1250752 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -30,6 +30,25 @@ set_target_properties(rnscreens PROPERTIES
     POSITION_INDEPENDENT_CODE ON
 )
 
+# FLAGS DE COMPILACIÓN (para C y C++)
+target_compile_options(
+    rnscreens 
+    PRIVATE
+    -ffile-prefix-map=${CMAKE_SOURCE_DIR}=.
+)
+
+# FLAGS DE LINKER
+target_link_options(
+    rnscreens 
+    PRIVATE
+    -Wl,--build-id=none
+    -Wl,--no-relax
+    -Wl,--pack-dyn-relocs=none
+    -Wl,--sort-common
+    -Wl,--sort-section=name
+    -Wl,--hash-style=gnu
+)
+
 target_compile_definitions(
     rnscreens
     PRIVATE
diff --git a/node_modules/react-native-screens/android/build.gradle b/node_modules/react-native-screens/android/build.gradle
index 872d44e..5cd90c6 100644
--- a/node_modules/react-native-screens/android/build.gradle
+++ b/node_modules/react-native-screens/android/build.gradle
@@ -128,10 +128,18 @@ android {
             abiFilters (*reactNativeArchitectures())
         }
         externalNativeBuild {
+            ndkBuild {
+                arguments "APP_CFLAGS+=-ffile-prefix-map=${rootDir}=.",
+                        "APP_CXXFLAGS+=-ffile-prefix-map=${rootDir}=.",
+                        "APP_LDFLAGS+=-Wl,--build-id=none -Wl,--no-relax -Wl,--pack-dyn-relocs=none -Wl,--sort-common -Wl,--sort-section=name -Wl,--hash-style=gnu"
+            }
             cmake {
                 arguments "-DANDROID_STL=c++_shared",
                         "-DRNS_NEW_ARCH_ENABLED=${IS_NEW_ARCHITECTURE_ENABLED}",
-                        "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
+                        "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON",
+                        "-DCMAKE_C_FLAGS=-ffile-prefix-map=${rootDir}=.",
+                        "-DCMAKE_CXX_FLAGS=-ffile-prefix-map=${rootDir}=.",
+                        "-DCMAKE_SHARED_LINKER_FLAGS=-Wl,--build-id=none -Wl,--no-relax -Wl,--pack-dyn-relocs=none -Wl,--sort-common -Wl,--sort-section=name -Wl,--hash-style=gnu"
             }
         }
     }
diff --git a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
index 8502124..04f3351 100644
--- a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
@@ -76,6 +76,19 @@ target_compile_options(
   -frtti
   -std=c++20
   -Wall
+  -ffile-prefix-map=${CMAKE_SOURCE_DIR}=.
+)
+
+# FLAGS DE LINKER
+target_link_options(
+  ${LIB_TARGET_NAME}
+  PRIVATE
+  -Wl,--build-id=none
+  -Wl,--no-relax
+  -Wl,--pack-dyn-relocs=none
+  -Wl,--sort-common
+  -Wl,--sort-section=name
+  -Wl,--hash-style=gnu
 )
 
 target_include_directories(
